apps/crn-server/test/data-providers/contentful/reminder.data-provider.test.ts

```diff
@@ -1377,10 +1386,46 @@ describe('Reminders data provider', () => {
           items: [researchOutputVersionItem],
         };
 
-        setContentfulMock(researchOutputVersionsCollection);
+        setContentfulMock({ researchOutputVersionsCollection });
         const result = await remindersDataProvider.fetch(fetchRemindersOptions);
         expect(result).toEqual({ total: 0, items: [] });
       });
+
+      test('Should not return published research output reminder, if there is a related version reminder', async () => {
+        const publishedResearchOutputItem =
+          getContentfulReminderResearchOutputCollectionItem();
+        publishedResearchOutputItem!.sys.publishedAt = publishedAt;
+        publishedResearchOutputItem!.addedDate = publishedAt;
+
+        publishedResearchOutputItem!.sys.id =
+          researchOutputVersionItem!.linkedFrom!.researchOutputsCollection!.items[0]!.sys.id;
+
+        const researchOutputVersionsCollection = {
+          items: [researchOutputVersionItem],
+        };
+
+        const researchOutputsCollection = {
+          items: [publishedResearchOutputItem],
+        };
+
+        setContentfulMock({
+          researchOutputVersionsCollection,
+          researchOutputsCollection,
+        });
+
+        const result = await remindersDataProvider.fetch(fetchRemindersOptions);
+        const expectedReminder = getResearchOutputVersionPublishedReminder();
+
+        expectedReminder.data.publishedAt = publishedAt;
+        expectedReminder.data.associationName = 'Working Group 1';
+        expectedReminder.data.associationType = 'working group';
+
+        expect(result.items.map((r) => r.type)).toContain('Published');
```

Comment:
this doesnt look like a Published reminder isn't returned, this expectation is the same as https://github.com/yldio/asap-hub/blob/1fff374f379f9afe07ee8d931abd89939297fce6/apps/crn-server/test/data-providers/contentful/reminder.data-provider.test.ts#L460 where the Publish reminder IS returned. I think you want to tell them apart by looking at the `entity` property